docs/guides/continuous-integration/circleci.mdx

```diff
@@ -11,14 +11,16 @@ title: CircleCI
 
 :::
 
-<!-- textlint-disable -->
-
-<DocsVideo src="https://youtube.com/embed/J-xbNtKgXfY" />
-
-<DocsVideo src="/img/snippets/running-in-ci.mp4" title="Cypress in CircleCI" />
-
 <!-- textlint-enable -->
 
+The
+[Cypress CircleCI Orb](https://circleci.com/developer/orbs/orb/cypress-io/cypress)
+is the _official_ CircleCI Orb of [Cypress.io](https://cypress.io). Although you
```

Comment:
I think linking to the marketing site when referring to Cypress isn't necessary.